Update Audio+ for changes to Tumblr's audio player

hobinjk opened the pull request.

hobinjk commented:
pls

nightpool commented:
Oh, dang, I don't remember seeing this pull at ALL

nightpool commented on Extensions/audio_plus.js:
```diff
@@ -100,116 +94,49 @@ XKit.extensions.audio_plus = {
 		var controls = audio_plus.pop_out_controls;
 		if (controls.classList.contains("playing")) {
-			audio_plus.stop_current_instance();
+			audio_plus.current_player.querySelector('audio').pause();
```
Would it be too annoying to register this handler when we start playing, so we can use closures instead of this ugly instance variable?

nightpool commented on Extensions/audio_plus.js:
```diff
@@ -259,19 +186,22 @@ XKit.extensions.audio_plus = {
 
-		var playing_players = document.querySelectorAll(".audio_player.playing");
-		if (playing_players.length === 0) {
+		// Check for pause icons within an audio player within a post
```
This is obvious enough that the comment adds clutter

hobinjk commented:
@nightpool I've removed the comment and renamed the variable to be more accurate. Unfortunately `current_player` is pretty necessary because the undocked controls need to know which player they're meant to be controlling.
